pkg/cloudprovider/providers/aws/aws_loadbalancer.go

```diff
@@ -62,6 +64,32 @@ func (s *AWSCloud) ensureLoadBalancer(namespacedName types.NamespacedName, loadB
 		if err != nil {
 			return nil, err
 		}
+
+		// XXX Do we always want to create this policy or should we do this lazily
+		// and create when requested
+		err = s.createProxyProtocolPolicy(loadBalancerName)
+		if err != nil {
+			return nil, err
+		}
+
+		if proxyProtocol {
+			for _, proxyListener := range listeners {
+				// TODO Refactor this out into a seperate method
+				request := &elb.SetLoadBalancerPoliciesForBackendServerInput{
```

Comment:
It is kind of annoying that we can't do this before creating the listener - I guess theoretically there's a window where we could have a listener without proxy-protocol?  Thankfully ELB is so slow to create I suspect this is entirely theoretical...